addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1750,6 +1750,9 @@ const Wysiwyg = Widget.extend({
             color = (eventName === "foreColor" ? 'text-' : 'bg-') + color;
         }
         const coloredElements = this.odooEditor.execCommand('applyColor', color, eventName === 'foreColor' ? 'color' : 'backgroundColor', this.lastMediaClicked);
+        if (!coloredElements || !coloredElements.length) {
+            return;
+        }
```

Comment:
I think you can remove this now, since the other PR was merged which adds these checks on line 1764.

Reply:
done!

Reply:
Thank you ! 👍